lib/web_ui/dev/test_runner.dart

```diff
@@ -117,7 +117,8 @@ class TestCommand extends Command<bool> with ArgUtils {
         return runIntegrationTests();
       case TestTypesRequested.all:
         // TODO(nurhan): https://github.com/flutter/flutter/issues/53322
-        if (runAllTests) {
+        // TODO(nurhan): Expand browser matrix for felt integration tests.
+        if (runAllTests && isChrome) {
```

Comment:
This will cause it to silently not run integration tests and report success. An unsuspecting contributor may mistakenly assume that all integration tests passed while in fact they didn't run (e.g. if on the command line you accidentally use firefox).

I'd prefer that we throw an error saying that integration tests are not supported for the chosen browser.

Reply:
This sugegstion won't work partially. As you would also remember, for Mac LUCI bots, we are running felt as:

```
felt tests --browser=safari 
```
It is in the LUCI recipe now. If we don't do this, we need to change the LUCI recipe (which will take a few days) I believe. **We can use luci.dart if you finished it?**

For firefox it is easy we can just change the code and say:

```
felt tests --browser=firefox --unit-tests-only
```

Reply:
Ah, I see. I guess I'll start working on `luci.dart` then :)

Reply:
thanks!